sql/privilege.md

```diff
@@ -289,7 +294,7 @@ When TiDB starts, some privilege-check tables are loaded into memory, and then t
 If an immediate effect is needed when you modify the `grant` table, you can run the following command:
 
 ```sql
-flush privileges
+flush privileges;
```

Comment:
Perhaps use ALL CAPS here

Reply:
@lilin90 do we have a style for SQL commands?  I like to write keywords in caps as kenny suggests, but many of our existing examples are lower case.

Reply:
If the style is all-lowercase then the `DROP USER 'test';` statement above should be changed to `drop user 'test';`.

Reply:
@lilin90 There are quite a few inconsistencies: `CREATE USER`, `REVOKE` are in caps.  I can go through and change once I have confirmation of the style.

Reply:
@morgo Thanks! Currently, we don't have a strict SQL style. But to improve the consistency, we can now establish one: "Capitalize all SQL keywords and reserved words." Later, I'll update that in docs-cn and tell other members to follow this style.

Reply:
@lilin90 thanks, I've updated all keywords to caps.  PTAL